app/api/exhibitors.py

```diff
@@ -30,13 +30,24 @@ def query(self, view_kwargs):
         query_ = event_query(query_, view_kwargs)
         return query_
 
+    def after_create_object(self, exhibitor, data, view_kwargs):
```

Comment:
Wrong, this is API is just for querying exhibitors. For creating them, ExhibitorListPost is present